app-emulation/virtualbox-additions/virtualbox-additions-7.0.2.ebuild

```diff
@@ -4,20 +4,15 @@
 EAPI=8
 
 MY_PN=VBoxGuestAdditions
-MY_PV="${PV/beta/BETA}"
-MY_PV="${MY_PV/rc/RC}"
-MY_P=${MY_PN}_${MY_PV}
+MY_P=${MY_PN}_${PV}
 
 DESCRIPTION="CD image containing guest additions for VirtualBox"
 HOMEPAGE="https://www.virtualbox.org/"
-SRC_URI="https://download.virtualbox.org/virtualbox/${MY_PV}/${MY_P}.iso"
+SRC_URI="https://download.virtualbox.org/virtualbox/${PV}/${MY_P}.iso"
 
-LICENSE="GPL-2+ LGPL-2.1+ MIT SGI-B-2.0 CDDL"
+LICENSE="GPL-3 CDDL"
 SLOT="0/$(ver_cut 1-2)"
-[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || \
-KEYWORDS="amd64"
-IUSE=""
-RESTRICT="mirror"
```

Comment:
This will fix https://bugs.gentoo.org/878457 btw (filed at a funny timing given this PR)

Reply:
I added Closes: to the commit message, but I didn't rename the PR to [please reassign] so it doesn't show

As a side note, it appeares upstream is trying to license virtualbox 7+ GPL-3 instead of GPL-2+, they updated COPYING and most of the source files, but they didn't update all the source files and the FAQ yet.  I updated LICENSE and will check on future bumps to see if they drop MIT or the other GPL licenses.

Reply:
Pop a note above LICENSE for that, then if no improvement in a few releases, send them a bug